js/Active-Links.js

```diff
@@ -1,26 +1,35 @@
-/* eslint-disable func-names */
-$(document).ready(() => {
-  $('.navbar-nav li a').click(function () {
-    $('.navbar-nav li a').parent().removeClass('active');
-    $(this).parent().addClass('active');
-  });
+(function () {
+  const $navbarLinksSelector = $('.navbar-nav li a'),
+    $window = $(window);
+
+  function onLinkClick() {
+    const $this = $(this);
+    $navbarLinksSelector.parent().removeClass('active');
+    $this.parent().addClass('active');
+  };
+
+  function highlightCurrLink() {
+    const $this = $(this),
+      $currSectionId = $this.attr('id'),
+      $currSectionHeight = $this.outerHeight(),
+      $currSectionPositionTop = $this.offset().top - 70,
+      $currSectionLinkSelector =
+        $(`.navbar-nav li a[href="#${$currSectionId}"]`);
 
-  $(window).scroll(() => {
-    $('section').each(function () {
-      const currSectionId = $(this).attr('id');
-      const currSectionHeight = $(this).outerHeight();
-      const currSectionPosition = $(this).offset().top - 70;
+    if(
+      $window.scrollTop() > $currSectionPositionTop
```

Comment:
use condition operator after line instead of before line 

`$window.scrollTop() > $currSectionPositionTop &&`